frameworks/digital-outcomes-and-specialists-5/manifests/declaration.yml

```diff
@@ -141,7 +141,7 @@
   name: Application accuracy
   slug: application-accuracy
   editable: True
-  prefill: False
+  prefill: True
```

Comment:
This had prefill disabled in DOS4 and DOS3 even though the question didn't change, I think because it doesn't make sense to prefill this confirmation.

Reply:
Good point. I'll add a comment to help us avoid this mistake next year.